include/metal/list/sort.hpp

```diff
@@ -46,9 +50,9 @@ namespace metal {
     ///
     /// ### See Also
     /// \see list, reverse, rotate
-    template<class seq, class lbd>
+    template<class seq, class... lbd>
     using sort = detail::call<
-        detail::_sort<lbd>::template type,
+        detail::_sort<lbd...>::template type,
```

Comment:
We [do already have default template arguments on aliases](https://github.com/brunocodutra/metal/blob/master/include/metal/value/value.hpp#L91), so there's hope.

Reply:
Do you mind if I open a couple of PRs in order to test different versions? Or should I prefer godbolt.org in order to reduce the load of your CI setup?

Reply:
I don't mind extra PRs.

Reply:
I tried, but I think it is not worth any more effort. I vote for the [workaround](https://github.com/brunocodutra/metal/pull/101#pullrequestreview-301046192).